libs/sdk-backend-tiger/src/convertors/fromBackend/HierarchyConverter.ts

```diff
@@ -0,0 +1,22 @@
+// (C) 2023 GoodData Corporation
+
+import { ObjectType } from "@gooddata/sdk-model";
+
+// TODO: where to put this interface?
+export interface IAttributeHierarchy {
+    id: string;
+    title: string;
+    description?: string;
+    attributes: { id: string; type: ObjectType }[];
```

Comment:
Should it be attributes: `ObjRef` or `IdRef[]` ? Interface should be likely in `sdk-model`, or do you have any concern why it should not be there?